src/topp/MultiplexResolver.cpp

```diff
@@ -455,18 +455,29 @@ class TOPPMultiplexResolver :
     
     for (ConsensusMap::ConstIterator cit = map_in.begin(); cit != map_in.end(); ++cit)
     {
+      //LOG_DEBUG << "consensus = " << (cit - map_in.begin());
+      //LOG_DEBUG << "       RT = " << cit->getRT();
+      //LOG_DEBUG << "       mz = " << cit->getMZ();
+      
+      // Consensus features without sequence annotations are written unchanged to the conflict output.
+      if (cit->getPeptideIdentifications().size() == 0)
+      {
+        //LOG_DEBUG << "  (no ID)\n\n";
+        
+        ConsensusFeature consensus(*cit);
+        map_conflicts.push_back(consensus);
+        
+        continue;
+      }
+      
       // extract the label set from the attached peptide sequence (There should be only one, since IDConflictResolver was run first.)
```

Comment:
Let's keep `IDConflictResolver` and `MultiplexResolver` separate.

In future, I want to take more than one sequence annotation into account. For example,  a SILAC triplet to which a light, medium and heavy ID were mapped is very reliable. Right now the tool considers only one (the best) ID.